styles/mixins.scss

```diff
@@ -0,0 +1,23 @@
+@mixin forPhoneDown {
```

Comment:
what do you thinks about naming this file `breakpoint.scss` instead of `mixins.scss`

Reply:
No. Not a good idea.